### ISSUE TITLE
row and column information

### ISSUE THREAD

**Issue by rubenmoor** [snoyberg/xml#67]
This is a feature request.

If I'm not mistaken, xml-conduit does not keep track of row and column information (e.g. in the nodes). My use case would benefit from something like that, because potential errors in the xml document are fed back to the user.

Any thoughts on that?

**Comment by snoyberg** [comment]
This is provided in the streaming API, e.g. `parseTextPos`. If you have a proposal for how to add it to the nodes as well, let me know.

**Comment by alexkalderimis** [comment]
I note that all the combinators operate on `Event`s rather than `EventPos` - is there a tidy way of using `Event` based parsing on `EventPos` sources?

**Comment by bitc** [comment]
What about html-conduit? `Text.HTML.Dom.eventConduit` gives `Events` and there is no equivalent that gives `EventPos`s

**Comment by snoyberg** [comment]
No, such a function doesn't exist, and we'd need the functionality added to tagstream-conduit first to make it happen.
